_opcodes/op022.html

```diff
@@ -50,8 +50,8 @@
 	<li>
 		<code>1</code> is the worst roll you can get, yet it is neither of the extremes. So, <code>-1</code> Luck is bad because it prevents you from bypassing the wait time, (rolling a <code>0</code>), but <b>if Luck must be negative, the lower the better</b>.
 		<ul>
-			<li><code>Luck = "-1"</code>: Worst roll possible is a <code>1</code>, you will be waiting for 7 AI updates (or, if you prefer, 7/15ths of a second).</li>
-			<li><code>Luck = "-7"</code>: Worst roll possible is a <code>7</code>, you will be waiting for 1 AI update (or, if you prefer, 1/15th of a second)!!</li>
+			<li><code>Luck = "-1"</code>: Worst roll possible is a <code>1</code>, you will be waiting for 7 AI updates (or, if you prefer, <small><sup>7</sup>&frasl;<sub>15</sub></small> ths of a second).</li>
+			<li><code>Luck = "-7"</code>: Worst roll possible is a <code>7</code>, you will be waiting for 1 AI update (or, if you prefer, <small><sup>1</sup>&frasl;<sub>15</sub></small> th of a second)!!</li>
```

Comment:
Shouldn't these have a dash or the space removed before the "th"?

Reply:
Added a dash.